x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -165,9 +165,10 @@ export const ExceptionListsTable = React.memo(() => {
   const handleExportSuccess = useCallback(
     (listId: string) =>
       (blob: Blob): void => {
+        addSuccess(i18n.EXCEPTION_EXPORT_SUCCESS);
         setExportDownload({ name: listId, blob });
       },
-    []
+    [addSuccess]
```

Comment:
@semd Why was this dependency removed in your pr? https://github.com/elastic/kibana/pull/103275/files#diff-23dad27ad5775a9231a0e8ab9d59dd337feae06d763ab45b63767f1101ee6059R168-R172